### PR TITLE
Update jetbrains2

### PR DESCRIPTION
**Pull request by dritter**
Hey @kouyk 

This fixes some issues with https://github.com/NixOS/nixpkgs/pull/201931 :
1. Update PhpStorm
2. Fix build for Goland on linux
3. Fix build for Idea-Ultimate on linux

### CODE REVIEW

File: pkgs/applications/editors/jetbrains/default.nix [hunk 1]
```diff
@@ -127,9 +127,9 @@ let
     }).overrideAttrs (attrs: {
       postFixup = (attrs.postFixup or "") + lib.optionalString stdenv.isLinux ''
         interp="$(cat $NIX_CC/nix-support/dynamic-linker)"
-        patchelf --set-interpreter $interp $out/goland*/plugins/go/lib/dlv/linux/dlv
+        patchelf --set-interpreter $interp $out/goland*/plugins/go-plugin/lib/dlv/linux*/dlv
 
-        chmod +x $out/goland*/plugins/go/lib/dlv/linux/dlv
+        chmod +x $out/goland*/plugins/go-plugin/lib/dlv/*/dlv
```
Comment (kouyk) [inline pkgs/applications/editors/jetbrains/default.nix]: Does the current packaging work for linux aarch64?

**Review by kouyk — COMMENTED** [review]
Thanks for looking into this, I was unavailable the past few days to test things out!

File: pkgs/applications/editors/jetbrains/default.nix (hunk 1 above)
Comment (kouyk) [reply pkgs/applications/editors/jetbrains/default.nix]: assuming that linux aarch64 works, should we patch just the relevant binary for the platform?

Comment (dritter) [reply pkgs/applications/editors/jetbrains/default.nix]: To be honest, I don't know if linux aarch64 works. But I trusted patchelf to do the right thing.. My guess was that if `linux/dlv` needed to be patched, than `linux-aarch64/dlv` needs to be patched as well. But I might be wrong.

Comment (dritter) [reply pkgs/applications/editors/jetbrains/default.nix]: I changed it now to patch only the relevant binary...

**Pull request closed without merging**